cmd/proxy/proxy.go

```diff
@@ -111,8 +111,8 @@ type Currency struct {
 }
 
 // GetParameters : Returns params from Currency struct
-func (k *Currency) GetParameters() (map[string]string) {
-	return k.params
+func (k *Currency) GetParameters() (map[string]string, error) {
```

Comment:
This plugin has to satisfy the `ServicePlugin` interface defined in https://github.com/katzenpost/server/blob/master/cborplugin/client.go#L63-L79. 

Keep this method as it is.

Reply:
I removed the changes but removed the error var waiting for the function call: https://github.com/hashcloak/Meson/pull/28/files#diff-3002d695552ec7e7ebbaaf04a21d3439L70-L74